webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+var webpack = require('webpack');
+var fs = require('fs');
+var copyright = fs.readFileSync('lib/copyright.frag', 'utf8');
+
+module.exports = {
+    entry: './lib/index.js',
+    output: {
+        path: path.join(__dirname, 'dist', 'browser'),
+        filename: 'OktaAuth.min.js',
```

Comment:
Is this the output fileName? If so, should this look something like okta-auth-js-{version}.min.js?